contract.md

```diff
@@ -12,29 +27,42 @@ like to work on, as part of your project (for example, other things related to D
+
 ## Teamwork plan
 
 _If you plan to work with someone else on the project, what is your plan for
 collaborating? What part(s) do you think you might do together? What part(s) are you
 considering doing separately? How will you hold one another accountable to make regular
 progress?_
 
+N/A I will be working by myself
+
 ## Critique plan
 
 _What will you do to make sure that you can give consistent and actionable feedback to
 other people in the class?_
 
+In order to make sure that I can give consistent and actionable feedback to other people in the class I will attend class on mondays and plan to talk with the people in class in which I am critiqueing. My plan will be to look over peoples journals before class starts in oder to see what they have accomplished with thier project so far/ since I last looked at it. Additionally in class I would like to have a quick meeting to discuss what they are planing to do for the following week and I will plan to also look over their code on github and leave feedback on it. I will plan to do this all on monday and if I become overwhelemed with work on a monday I will finish the critique sepcifically code reviews on tuesday. I belive that if I stay ontop of a schedual like this every week, I should be able to provide good, consistent and actionable feedback to people in the class. 
+
+
 ## Success
 
 _At the end of the semester, what would you be proud to show or tell someone about your
 project?_
 
+At the end of the semester I would be proud to show/tell someone that I built A DSL for live stat keeping that is simple and does not require pen and paper or an extensive excell sheet. I would esspecially be proud to tell my dad that he no longer needs to bring a notepad for stat keeping to my brothers football games. If this project goes really well I think it could have potential aplications within the highschool and collegiate football realm. What I mean by this is that currently coaches at the highschool and lower colligiate levels have to manually input stats for their players into their systems. So what if instead they had a DSL that allowed them to keep live stat keeping that could hook into thier systems and manually update in live time. Although I do not think It would be possible to expand to the that level over the last few weeks of the semester, I think it would be really cool to show proof of concept such as having the dsl update a spreadsheet in excell live. Overall I would be most proud to show off the language weather it is more natural langauge esc or a combination of natural language and method based, also I would be very proud to show that a simple dsl can build a stat sheet for an entire team and the potential to activly update an excell sheet.
+
```

Comment:
I really like that you have a more personal motivation behind this project! It shows that you relate more to the user's experience and that you care about the impact of the work you are doing.